lib/snowflake_db/connection.rb

```diff
@@ -0,0 +1,56 @@
+require 'odbc_utf8'
+require 'sequel'
+module SnowflakeDB
+  class Connection
+    attr_reader :config, :unload_target, :max_file_size
+
+    PG_INCLUDE_KEYS = %w(host dbname port user password)
+    MAX_FILE_SIZE = (5*1024*1024*1024).freeze
```

Comment:
activesupport is included in the project, so this should just be able to be expressed as `5.gigabytes`